apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -852,10 +855,12 @@ const Ticket = new GQLListSchema('Ticket', {
             const userType = get(user, 'type')
 
             if (userType === RESIDENT && operation === 'create') {
+                const locale = extractReqLocale(context.req) || conf.DEFAULT_LOCALE
                 await checkDailyTicketLimit({
                     userId: get(user, 'id'),
                     organizationId: get(resolvedData, 'organization'),
                     details: get(resolvedData, 'details'),
+                    isInvoiceTicket: get(resolvedData, 'details') === i18n('marketplace.invoice.newTicket.details', { locale }),
```

Comment:
Should the ticket have the "isPayable" flag selected? Can we add this condition here?

Reply:
yes, you're right, you can additionally secure this check